cairo/src/font/font_face.rs

```diff
@@ -64,9 +64,7 @@ impl FontFace {
     /// `FontFace` is dropped.
     #[cfg(any(feature = "freetype", feature = "dox"))]
     #[doc(alias = "cairo_ft_font_face_create_for_ft_face")]
-    pub unsafe fn create_from_ft(
```

Comment:
I am not sure if it is a great idea to use the ffi bindings here

Reply:
I'm not sure that changing to a safe method is an appropriate change, it's a big change.